src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -6455,4 +6457,12 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="IDS_FeatureExtensionGetEnumerator" xml:space="preserve">
     <value>extension GetEnumerator</value>
   </data>
+  <data name="WRN_GeneratorFailedDuringGeneration_Description" xml:space="preserve">
```

Comment:
>WRN_GeneratorFailedDuringGeneration_Description [](start = 14, length = 47)

Consider moving these next to the corresponding warnings so the relationship is clear.